### PR TITLE
[ci-visibility] Separate excluded branches and tracked files into different sections

### PR DESCRIPTION
**Pull request by juan-fernandez**
### What does this PR do? What is the motivation?
Make it easier to discover tracked files feature. 

### Merge instructions

- [x] Please merge after reviewing

### Additional notes

Preview: https://docs-staging.datadoghq.com/juan-fernandez/organize-tracked-files/continuous_integration/intelligent_test_runner/#configuration

### CODE REVIEW

File: content/en/continuous_integration/intelligent_test_runner/_index.md [hunk 1]
```diff
@@ -18,17 +18,17 @@ Intelligent Test Runner is Datadog's test impact analysis solution. It automatic
...
+By minimizing the number of tests run per commit, Intelligent Test Runner reduces the frequency of [flaky tests][1] disrupting your pipelines. This can be particularly frustrating when the test flaking is unrelated to the code change being tested. After enabling Intelligent Test Runner for your test services, you can limit each commit to its relevant tests to ensure that flaky tests unrelated to your code change don't end up arbitrarily breaking your build.
 
 ### Limitations
 
 There are known limitations in the current implementation of Intelligent Test Runner that can cause it to skip tests that should be run under certain conditions. Specifically, Intelligent Test Runner is not able to detect changes in library dependencies, compiler options, external services or changes to data files in data-driven tests.
 
-To override Intelligent Test Runner and run all tests, add `ITR:NoSkip` (case insensitive) anywhere in your Git commit message.
+To override Intelligent Test Runner and run all tests, add `ITR:NoSkip` (case insensitive) anywhere in your Git commit message. It is also possible to add a list of [excluded branches](#excluded-branches) and [tracked files](#tracked-files).
```
Comment (drichards-87) [inline content/en/continuous_integration/intelligent_test_runner/_index.md:27]:
```suggestion
To override Intelligent Test Runner and run all tests, add `ITR:NoSkip` (case insensitive) anywhere in your Git commit message. You can also add a list of [excluded branches](#excluded-branches) and [tracked files](#tracked-files).
```

File: content/en/continuous_integration/intelligent_test_runner/_index.md [hunk 2]
```diff
@@ -18,17 +18,17 @@ Intelligent Test Runner is Datadog's test impact analysis solution. It automatic
...
 
 ## Set up a Datadog library
 
-Prior to setting up Intelligent Test Runner, you must set up [Test Visibility][1] for your particular language. If you are reporting data through the Agent, use v6.40 or 7.40 and later.
+Prior to setting up Intelligent Test Runner, you must set up [Test Visibility][2] for your particular language. If you are reporting data through the Agent, use v6.40 or 7.40 and later.
```
Comment (drichards-87) [inline content/en/continuous_integration/intelligent_test_runner/_index.md:31]:
```suggestion
Before setting up Intelligent Test Runner, you must configure [Test Visibility][2] for your particular language. If you are reporting data through the Agent, use v6.40 or 7.40 and later.
```

**Review by drichards-87 — CHANGES_REQUESTED** [review]
Left you two small suggested edits.

**Comment by juan-fernandez** [comment]
> Left you two small suggested edits.

thanks! I applied all the suggestions

**Review by drichards-87 — APPROVED** [review]

**Pull request merged**